src/tools/kdb/mv.cpp

```diff
@@ -75,6 +75,12 @@ int MvCommand::execute (Cmdline const & cl)
 			return 11;
 		}
 		newConf.append (rename_key (k, sourceName, newDirName, cl.verbose));
+
+		// move the remaining keys back to the original position
+		while ((k = oldConf.next ()))
+		{
+			newConf.append (k);
+		}
```

Comment:
```suggestion
		for (k : oldConf)
		{
			newConf.append (k);
		}
```

I think this should work, once you rebase you should have an example to work off a few lines above (the other branch of `if(cl.recursive)`).